.github/workflows/push.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/checkout@master
       - uses: actions/setup-node@v4
         with:
-          node-version: 16.x
+          node-version: 18.x
```

Comment:
We can do `^20.15.0` as this is the version `yarn@3.8.2` in `package.json` uses.

Reply:
Done